packages/synapse-react-client/src/components/CardContainerLogic/CardContainerLogic.stories.ts

```diff
@@ -101,3 +112,76 @@ export const PublicationCard: Story = {
     genericCardSchema: publicationSchema,
   },
 }
+
+const statsConfig: StatConfig[] = [
+  { columnName: 'countPatients', label: 'Patients' },
+  { columnName: 'countSamples', label: 'Samples' },
+]
+
+const currentReleaseCardSql = `SELECT * FROM ${MOCK_RELEASE_CARDS_TABLE_ID} WHERE isCurrentRelease = true`
+
+export const ReleaseCardLargeMock: Story = {
+  args: {
+    sql: currentReleaseCardSql,
+    type: RELEASE_CARD,
+    limit: 3,
+    releaseCardConfig: {
+      cardSize: 'large',
+      prependRelease: false,
+      statsConfig: statsConfig,
+      buttonToExplorePageConfig: {
+        label: 'Explore Current Data Release',
+        sourcePathColumnName: 'releaseExplorePath',
+        exploreDataSql: currentReleaseCardSql, // generally this would refer to a different table, not the source table
+        exploreDataFacetColumnName: 'releaseType',
+        sourceDataFacetValueColumnName: 'releaseType',
+      },
+      dataGuidePath: 'data guide',
+    },
+  },
+  parameters: {
```

Comment:
So the storybook user doesn't have to explicitly switch to the mock stack
```suggestion
  parameters: {
    stack: 'mock',
```